### PR TITLE
[increment patch] Add basic localization setup

### PR DESCRIPTION
**Pull request by zinabat**
- Loads `en` locale from file
- Preparation for adding Serge
- This PR also tests Travis will publish to `npm`

### CODE REVIEW

File: lang/locales/en.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/* eslint quotes: ["error", "double"] */
+export const en = {
+	"actions": "Actions",
```
Comment (m6jones) [inline lang/locales/en.js:3]: You need context comments after each of these :)

Comment (zinabat) [reply lang/locales/en.js]: Dun!

Comment (AKobets) [reply lang/locales/en.js]: I think that lang files should be added to packaged files list in package.json

**Comment by AKobets** [comment]
I think that lang files should be added to packaged files list in package.json

**Comment by zinabat** [comment]
This PR for some reason is not pulling in branch changes.

**Pull request closed without merging**